ansible/ansible.cfg

```diff
@@ -1,11 +1,4 @@
 [defaults]
-
-# Automatically accept SSH thumbprints when machines are provisioned.
+vault_password_file = vault.password
 host_key_checking = False
```

Comment:
I'd add that comment back to `host_key_checking` and also mention in it that this option should never be used in production, as it allows man-in-the-middle attacks.

Reply:
Done.